gocd/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: gocd
 home: https://www.gocd.org/
-version: 2.12.1
+version: 2.13.0
```

Comment:
As currently set up, this would have to be `3.0.0` but I'd prefer to make it backward compatible, see suggestions below.

Reply:
Sure